### PR TITLE
review Mandarin translation for README.md in PR #3172 (close #3170) 

### LINKED ISSUE #3170: Translation of README.md into Mandarin
I was hoping that translating the README file to the most spoken language in the world, Mandarin would help with the outreach of the repository. In addition, I would like to work on this as a part of Hacktoberfest.

### PR DESCRIPTION
**Pull request by johnbanq**
### Description
This is a follow-up of PR #3172, it contains a reviewed version of the mandarin translation done by @jagreetdg
 
### Affected components
- [ ] Docs

### Related Issues
#3170

### Note
Despite having revised it multiple times, there might still be some overlooked issues. It is suggested to get another native mandarin speaker to do a quick check, but should be OK to settle on this version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,12 +214,13 @@ This readme is available in the following translations:
 
 - [Japanese :jp:](translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
 - [French :fr:](translations/README.french.md) (:pray: [@l0ck3](https://github.com/l0ck3))
+- [Chinese :cn:](translations/README.chinese.md) (:pray: [@jagreetdg](https://github.com/jagreetdg) & [@johnbanq](https://github.com/johnbanq))
 - [Bosnian :bosnia_herzegovina:](translations/README.bosnian.md) (:pray: [@hajro92](https://github.com/hajro92))
 - [Russian :ru:](translations/README.russian.md) (:pray: [@highflyer910](https://github.com/highflyer910))
-- [Greek 🇬🇷](translations/README.greek.md) (:pray: [@MIP2000](https://github.com/MIP2000))
-- [Spanish 🇲🇽](/translations/README.mx_spanish.md)(:pray: [@ferdox2](https://github.com/ferdox2))
+- [Greek đŹđˇ](translations/README.greek.md) (:pray: [@MIP2000](https://github.com/MIP2000))
```
Comment (marionschleifer) [inline README.md]: Something went wrong with the Greek, Spanish & German flags here:

image

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Read more at [hasura.io](https://hasura.io) and the [docs](https://docs.hasura.i
 * **Fine-grained access control**: Dynamic access control that integrates with your auth system (eg: auth0, firebase-auth)
 * **High-performance & low-footprint**: ~15MB docker image; ~50MB RAM @ 1000 req/s; multi-core aware
 * **Admin UI & Migrations**: Admin UI & Rails-inspired schema migrations
-* **Postgres** ❤️: Supports Postgres types (PostGIS/geo-location, etc.), turns views to *graphs*, trigger stored functions or procedures with mutations
+* **Postgres** â¤ď¸: Supports Postgres types (PostGIS/geo-location, etc.), turns views to *graphs*, trigger stored functions or procedures with mutations
```
Comment (marionschleifer) [inline README.md]: The heart doesn't show up here anymore:

image

File: README.md [hunk 3]
```diff
@@ -180,7 +180,7 @@ Check out our [contributing guide](CONTRIBUTING.md) for more details.
 Hasura brand assets (logos, the Hasura mascot, powered by badges etc.) can be
 found in the [assets/brand](assets/brand) folder. Feel free to use them in your
 application/website etc. We'd be thrilled if you add the "Powered by Hasura"
-badge to your applications built using Hasura. ❤️
+badge to your applications built using Hasura. â¤ď¸
```
Comment (marionschleifer) [inline README.md]: Same here.

File: translations/README.chinese.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+**目录**
+
+- [快速开始](#快速开始)
+    - [一键部署到Heroku](#一键部署到Heroku)
+    - [其它部署方法](#其它部署方法)
+- [架构](#架构)
+- [客户端工具](#客户端工具)
+- [添加业务逻辑](#添加业务逻辑)
+    - [远端Schema](#远端Schema)
+    - [通过数据库事件调用Webhook](#通过数据库事件触发Webhooks)
```
Comment (marionschleifer) [inline translations/README.chinese.md]: This link doesn't work.

File: translations/README.chinese.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+如果您要报告安全问题，请参考[该文档](../SECURITY.md).
+
+## 贡献
+
+查阅我们的[贡献指南](../CONTRIBUTING.md) 了解更多细节.
+
+## 品牌资料
+
+你可以在[assets/brand](assets/brand)文件夹中找到Hasura的品牌资产（logo，Hasura吉祥物，powered-by badges等）。请随意在你的应用程序/网站等地方使用它们吧！我们将非常高兴看到在你用Hasura构建的应用里看到"Powered by Hasura"徽章。❤️
```
Comment (marionschleifer) [inline translations/README.chinese.md]: This link doesn't work.

File: translations/README.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md)  [:bosnia_herzegovina: Bosnian](README.bosnian.md) |
+| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md)  [:bosnia_herzegovina: Bosnian](README.bosnian.md) | [:cn: 中文](README.chinese.md) |
```
Comment (marionschleifer) [inline translations/README.md]: The Chinese translation doesn't show here:

image

**Comment by johnbanq** [comment]
It seems to be a problem in encoding. Is the README in UTF-8? VScode shows it is encoded in ISO8859-2. I am not sure in what format to save

**Comment by shahidhk** [comment]
@johnbanq Encoding should be UTF-8

**Comment by shahidhk** [comment]
Closed in favour of https://github.com/hasura/graphql-engine/pull/3639 as I dont have permission to push to this repo.

**Pull request closed without merging**